### PR TITLE
Arturia.ArturiaSoftwareCenter version 2.4.5.760

### PR DESCRIPTION
**Pull request by chgure**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/97063)

### CODE REVIEW

**Comment by chgure** [comment]
@microsoft-github-policy-service agree

File: manifests/a/Arturia/ArturiaSoftwareCenter/2.4.5.760/Arturia.ArturiaSoftwareCenter.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Arturia.ArturiaSoftwareCenter
+PackageVersion: 2.4.5.760
```
Comment (mdanish-kh) [inline manifests/a/Arturia/ArturiaSoftwareCenter/2.4.5.760/Arturia.ArturiaSoftwareCenter.installer.yaml:5]:
```suggestion
PackageVersion: 2.4.5
```
Must match the version shown in Control Panel / Apps&Features

File: manifests/a/Arturia/ArturiaSoftwareCenter/2.4.5.760/Arturia.ArturiaSoftwareCenter.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Arturia.ArturiaSoftwareCenter
+PackageVersion: 2.4.5.760
```
Comment (mdanish-kh) [inline manifests/a/Arturia/ArturiaSoftwareCenter/2.4.5.760/Arturia.ArturiaSoftwareCenter.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 2.4.5
```

File: manifests/a/Arturia/ArturiaSoftwareCenter/2.4.5.760/Arturia.ArturiaSoftwareCenter.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.4.0.schema.json
+
+PackageIdentifier: Arturia.ArturiaSoftwareCenter
+PackageVersion: 2.4.5.760
```
Comment (mdanish-kh) [inline manifests/a/Arturia/ArturiaSoftwareCenter/2.4.5.760/Arturia.ArturiaSoftwareCenter.yaml:5]:
```suggestion
PackageVersion: 2.4.5
```
Must match the version shown in Control Panel / Apps&Features

File: manifests/a/Arturia/ArturiaSoftwareCenter/2.4.5.760/Arturia.ArturiaSoftwareCenter.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Arturia.ArturiaSoftwareCenter
+PackageVersion: 2.4.5.760
+PackageLocale: en-US
+Publisher: Arturia
+PackageName: Arturia Software Center
+License: Proprietary
+Copyright: (c) 2014 Arturia
+ShortDescription: Arturia Software Center Setup
```
Comment (mdanish-kh) [inline manifests/a/Arturia/ArturiaSoftwareCenter/2.4.5.760/Arturia.ArturiaSoftwareCenter.locale.en-US.yaml:11]:
```suggestion
ShortDescription: The Arturia Software Center · Is an elegant, simple solution to help you install, activate, and update your Arturia software instruments.
```
ShortDescription should be more descriptive

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
After committing the suggested changes, you would also have to rename the version folder name to match the new PackageVersion. The folder under `manifests/a/Arturia/ArturiaSoftwareCenter` should be renamed from `2.4.5.760` -> `2.4.5`.

A quick and easy way to do this would be here in GitHub Web by pressing the <kbd>.</kbd> (dot) key on your keyboard that'll open up the PR in VS Code Online. Navigate to Explorer menu, find the required files and make the appropriate changes.

After that, stage and commit your changes from Source Control menu on the left. (The changes are automatically pushed)

**Comment by stephengillie** [comment]
No recent activity.

**Pull request closed without merging**